### PR TITLE
docs: add auto-translate workflow guide

### PR DESCRIPTION
**Pull request by sidemt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

This is a guide for the i18n team about the new auto-translation GitHub Actions workflow.

### CODE REVIEW

File: src/content/docs/language-lead-handbook.mdx [hunk 1]
```diff
@@ -40,6 +40,86 @@ The original author and the original article are linked automatically adding thi
...
+
+3. Click "Get started" button for your language to open the issue template
+
+The minimum requirements to make the workflow work are as follows:
+
+- 2-letter language code in the square brackets at the beginning of the issue title
+- URL of the English article in the description as a link
+- `auto` label
+
+![Issue templage](../../../public/images/github/auto-translate-issue.png)
```
Comment (majestic-owl448) [inline src/content/docs/language-lead-handbook.mdx]:
```suggestion
![Issue template](../../../public/images/github/auto-translate-issue.png)
```

File: src/content/docs/language-lead-handbook.mdx [hunk 2]
```diff
@@ -40,6 +40,86 @@ The original author and the original article are linked automatically adding thi
 
 With `link` being the link of the original article.
 
+## How to set up the auto-translation workflow for a new language
+
+To set up the auto-translation workflow for your language, you need to:
+
```
Comment (ojeytonwilliams) [inline src/content/docs/language-lead-handbook.mdx]:
```suggestion
```
This mostly repeats the heading, so I don't think it's necessary.

File: src/content/docs/language-lead-handbook.mdx [hunk 3]
```diff
@@ -40,6 +40,86 @@ The original author and the original article are linked automatically adding thi
...
+In this workflow, each GitHub issue corresponds to an article to be translated.
+
+1. Go to [freeCodeCamp/news-translation-tasks](https://github.com/freeCodeCamp/news-translation-tasks) repository
+
+2. Click "New issue"
+
+3. Click "Get started" button for your language to open the issue template
+
+The minimum requirements to make the workflow work are as follows:
```
Comment (ojeytonwilliams) [inline src/content/docs/language-lead-handbook.mdx]:
```suggestion
The minimum requirements to use the workflow are as follows:
```
Bit verbose.

**Review by ojeytonwilliams — COMMENTED** [review]
Looks good! The following are just nitpicks:

**Comment by sidemt** [comment]
Thanks for your feedback, @ilenia-magoni and @ojeytonwilliams !

**Comment by ojeytonwilliams** [comment]
Thanks, Yoko. I'm not in a position to judge if these are correct, so I won't leave an approval, but I do think the language is clear!

**Pull request merged**